personal/wojciech_smolak/R/cw1.R

```diff
@@ -0,0 +1,58 @@
+library("sf")
+library("raster")
+library("spData")
+library("spDataLarge")
+library(dplyr)
+library(ggplot2)
+
+names(world)
+class(world)
+plot(world)
+
+summary(world)
+summary(world["lifeExp"]) 
+
+world_mini <- world[1:2,1:3]
+world_mini
+
+
+# Wy�wietlcie map� kraj�w Europejskich  wg liczby ludno�ci
```

Comment:
zwracaj uwagę, czy w komentarzach pojawiają się dziwne znaczki. To może świadczyć o źle zapisanym kodowaniu znaków. Najbezpieczniej zrezygnować z polskich znaków, lub pisać od początku wszystko w języku angielskim